bin/logos.pl

```diff
@@ -566,6 +566,8 @@ package main;
 		} elsif($line =~ /\G%property\s*(?:\((\s*\w+\s*(?:,\s*(?:\w|\=|:)+\s*)*)\))?\s*((?:\w+\s+\**)+)(\w+)\s*;/gc){
 			nestingMustContain($lineno, "%property", \@nestingstack, "hook", "subclass");
 
+			$currentClass->hasinstancehooks(1);
```

Comment:
Is this line required if the edit on line 101 in bin/lib/Logos/Class.pm is in place?